Update intro to mention teach-rs (#124)
tweedegolf/rust-training: book/src/introduction.md

<a href="/slides/1_1-introduction/" target="_blank">Slides</a>

This is the open-source Rust training material we use for our professional multi-day (embedded) Rust training programs.

It comprises several modules based on the material of the [teach-rs](https://trifectatech.org/initiatives/education/) project (now part of [Trifecta Tech Foundation](https://trifectatech.org)) that we previously co-developed and that we contribute back to when improvements are made.

Published under the [CC-BY-SA 4.0 license](https://creativecommons.org/licenses/by-sa/4.0/), you
are free to use and share this material as long as you give us appropriate credit and share any modifications you make under that same license.

> If you spot any mistake or have a suggestion, please open an issue or pull request on [the GitHub repository](https://github.com/tweedegolf/rust-training).

Reviewer: ```suggestion
If you spot any mistakes or have a suggestion, please open an issue or pull request on [the GitHub repository](https://github.com/tweedegolf/rust-training).
```

## Let us train you!

> Studying on your own is good, but being guided with your team through an interactive tailor-made program is... well, better! (And a lot more fun as well!)

Reviewer: ```suggestion
Studying on your own is good, but being guided through an interactive tailor-made program with your team is more productive, and more fun!
```
